### PR TITLE
Redux persistent middleware

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**

- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/redux-persist/index.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (aabidsofi19) [inline src/redux-persist/index.tsx:1]: @sudhanshutech  dont skip  ts checks

**Review by aabidsofi19 — APPROVED** [review]

**Comment by sudhanshutech** [comment]
@aabidsofi19 
 i might need help with that redux middleware. Its building fine in sistent but could be due some package conflicts or breaking changes around @type/react18  its breaking while testing in cloud or meshery .
 Some unrelated errors coming .

**Comment by sudhanshutech** [comment]
@captain-Akshay

File: src/redux-persist/index.tsx (hunk 1 above)
Comment (leecalcote) [reply src/redux-persist/index.tsx:1]: @sudhanshutech does the work?

**Comment by captain-Akshay** [comment]
It is not building on mine @sudhanshutech

**Pull request closed without merging**